modules/ROOT/partials/configuration/uploadcare_store_type.adoc

```diff
@@ -0,0 +1,29 @@
+[[uploadcare-store-type]]
+== `uploadcare_store_type`
+
+Specifies the storage type for uploaded files. Use this option to control whether files are stored temporarily or permanently, or allow Uploadcare to decide automatically based on your account settings.
+
+* `auto`: Default behavior; Uploadcare decides the storage type.
```

Comment:
We hard coded it to permanent in the pluign. Since that is the most likely scenario for our users. I mostly added this option since I was unsure if we needed it for demo purposes.

```suggestion
* `auto`: Uploadcare decides the storage type.
```